performance-tools/performance-cluster/scripts/helpers/deprovision-gke-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
That script is an exact copy of `/prow/scripts/cluster-integration/helpers/deprovision-gke-cluster.sh`.
Also, I saw that there are other scripts that are very similar to existing ones for example `create-image.sh`
Is there a reason to not reuse existing scripts?

Reply:
Hi @kfurgol , @a-thaler asked for having all the related tools for perfomance test in test-infra. And **these scripts are not for the creation of prow jobs.** Some of them have been adjusted for a different usage others keep the same logic. @Piotr Bochynski is aware of that as we communicated in that kyma-perf-corner meeting.